test/go-tests/graceful_shutdown_test.go

```diff
@@ -117,33 +116,39 @@ func Test_GracefulShutdown(t *testing.T) {
 
 	t.Log("Verify Direct delivery of helloservice in stage dev")
 	err = VerifyDirectDeployment(serviceName, keptnProjectName, "dev", "ghcr.io/podtato-head/podtatoserver", "v0.1.0")
-	require.Nil(t, err)
+	logError(err, t, shipyardPod)
 
 	t.Log("Verify network access to public URI of helloservice in stage dev")
 	cartPubURL, err := GetPublicURLOfService(serviceName, keptnProjectName, "dev")
-	require.Nil(t, err)
+	logError(err, t, shipyardPod)
+
 	err = WaitForURL(cartPubURL+serviceHealthCheckEndpoint, time.Minute)
-	require.Nil(t, err)
+	logError(err, t, shipyardPod)
 
 	t.Log("Verify delivery of helloservice:v0.1.0 in stage staging")
 	err = VerifyBlueGreenDeployment(serviceName, keptnProjectName, "staging", "ghcr.io/podtato-head/podtatoserver", "v0.1.0")
-	require.Nil(t, err)
+	logError(err, t, shipyardPod)
 
 	t.Log("Verify network access to public URI of helloservice in stage staging")
 	cartPubURL, err = GetPublicURLOfService(serviceName, keptnProjectName, "staging")
-	require.Nil(t, err)
+	logError(err, t, shipyardPod)
+
 	err = WaitForURL(cartPubURL+serviceHealthCheckEndpoint, time.Minute)
-	if err != nil {
-		t.Log(GetDiagnostics(shipyardPod, ""))
-	}
-	require.Nil(t, err)
```

Comment:
No `require.Nil` at the end of the function even?

Reply:
require nil is included in logError !

Reply:
aha ok then :)